multiqc/modules/picard/RnaSeqMetrics.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+
+""" MultiQC submodule to parse output from Picard RnaSeqMetrics """
+
+from collections import OrderedDict, defaultdict
+import logging
+import os
+import re
+
+from multiqc import config, plots
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+DESC = {
```

Comment:
I think this is a placeholder to hold the long descriptions for each statistic, copied from the Picard website.